### DISCUSSION TITLE
Make autogenerated views more customizable

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5635] [Feature Requests]
Full customizability is not required, but a basic option to specify which fields are included would be great.

This would help in cases where you want to display a field that is not present by default, or want to hide unused fields.

This feature might also allow reordering fields.

**Comment by grantfitzsimmons** [comment]
This would be best done with a WYSIWYG editor. Maybe we are overestimating what users want to do for the most part, and we should take a step back.

Really really really rough draft simple implementation I discussed with @maxpatiiuk months ago (excerpt of our conversation):

> ![image](https://github.com/specify/specify7/assets/37256050/e6ea04e7-ea95-4f58-b8e2-ed86332c6ad9)
> 
> **Max:**
> Where should it be stored?
> User pref, collection pref, institution pref, something else?
> 
> **Me:**
> Good question. Ideally, in its own resource under the current user that could be copied to other levels. It would also need to be restricted by the ability to create app resources in that case.
> 
> **Max:**
> The simplified visual editor would probably be a full day of work for me (with no interruptions) - that’s considering it can just use collection prefs for storage - if it has to also get a new storage system, then a bit longer (creating another thing that has all the functionality of collection prefs but is stored as a separate app resource is trivial, but if storage requires some more special functionality, it gets harder)
